jobs/e2e_node/crio/latest/image-config-cgroupv1-resource-managers.yaml

```diff
@@ -4,4 +4,4 @@ images:
     project: fedora-coreos-cloud
     # Using `n1-standard-4` to enable CPU manager node e2e tests.
     machine: n1-standard-4
-    metadata: "user-data</home/prow/go/src/k8s.io/test-infra/jobs/e2e_node/crio/crio_cgroupsv2.ign"
+    metadata: "user-data</home/prow/go/src/k8s.io/test-infra/jobs/e2e_node/crio/crio_cgroupv1.ign"
```

Comment:
Same here. Is this v1 or v2?

Reply:
See https://github.com/kubernetes/test-infra/pull/33374#discussion_r1732796002